Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/WebViewControlProcess.cs

```diff
@@ -99,7 +97,7 @@ internal async Task<WebViewControlHost> CreateWebViewControlHostAsync(IntPtr hos
             return new WebViewControlHost(wvc);
         }
 
-        private IAsyncOperation<WebViewControl> CreateWebViewControlAsync(
+        private IAsyncOperation<Windows.Web.UI.Interop.WebViewControl> CreateWebViewControlAsync(
```

Comment:
Why ❔

Reply:
That's gotta be a search & replace that got scoped wrong somewhere & I didn't notice.